raiden/tests/integration/api/test_pythonapi_regression.py

```diff
@@ -40,14 +40,21 @@ def run_test_close_regression(raiden_network, deposit, token_addresses):
     # Initialize app2 balance proof and close the channel
     amount = 10
     identifier = 42
+    secret = factories.make_secret()
     assert api1.transfer(
-        registry_address, token_address, amount, api2.address, identifier=identifier
+        registry_address, token_address, amount, api2.address, identifier=identifier, secret=secret
```

Comment:
why are you giving the secret explicitly? I don't see a big problem, but I also don't see any benefit.